src/app.js

```
@@ -1 +1,76 @@
+/* eslint-disable no-console */
 // write code here
+'use strict';
+
+const fs = require('fs/promises');
+const path = require('path');
+
+async function move(oldPath, newPath) {
+  if (!oldPath || !newPath) {
+    console.error('Source and destination are required.');
+
+    return;
+  }
+
+  if (oldPath === newPath) {
+    return;
+  }
```

Minor: when `oldPath === newPath` the function silently returns. That's acceptable, but consider whether you want to print a message or exit with a non-zero code depending on how strict you need to be. The task description doesn't require output in this case, so this is optional.